src/components/selections/attributes.js

```diff
@@ -54,6 +54,29 @@ module.exports = overrideAll(templatedArray('selection', {
         description: 'Sets the selection\'s end y position.'
     },
 
+    xshift: {
+        valType: 'number',
+        dflt: 0,
+        min: -0.5,
+        max: 0.5,
```

Comment:
We could/should allow ranges beyond [-0.5, 0.5] so that one could highlight a histogram or a bar with `width: 1` using a value like 0.7.

Reply:
done 👍